CHANGELOG.md

```diff
@@ -580,7 +580,7 @@ of the `"go.opentelemetry.io/otel/bridge/opentracing".BridgeTracer` type. (#2911
 - The `LabelFilterSelector` interface from `go.opentelemetry.io/otel/sdk/metric/processor/reducer` is renamed to `AttributeFilterSelector`.
   The method included in the renamed interface also changed from `LabelFilterFor` to `AttributeFilterFor`. (#2790)
 - The `Metadata.Labels` method from the `go.opentelemetry.io/otel/sdk/metric/export` package is renamed to `Metadata.Attributes`.
-  Consequentially, the `Record` type from the same package also has had the embedded method renamed. (#2790)
+  Consequently, the `Record` type from the same package also has had the embedded method renamed. (#2790)
```

Comment:
I'm, not sure Consequentially is wrong here (nor would it be everywhere else)

Reply:
I can revert these :)